### DISCUSSION TITLE
Monitor systemd units in the docker version?

### DISCUSSION THREAD

**Discussion by bl4ckswordsman** [henrygd/beszel#327] [Ideas]
If I'm not mistaken, when running the docker version, Beszel currently supports showing docker containers info in the CPU and RAM usage graphs. Not systemd services.

How complicated do you think it would be implementing support for this?
I know another monitoring tool called Netdata supports this (they're obviously more staffed, [~20 paid devs corp](https://www.netdata.cloud/blog/redefining-monitoring-with-netdata/#:~:text=make%20them%2020%20by%20the%20end%20of%202020)). And for it to work, it's required to mount `/run/dbus`.

Personally, this is the only thing I'm missing in an otherwise perfect monitoring service.
Anyways, thanks a lot for building a great tool!

EDIT: I meant to say "processes outside docker" more than systemd units. But both would be cool.

**Comment by henrygd** [comment]
I haven't looked into this yet, but I agree it would be a nice feature if it can implemented cleanly. I'll add it to my list.

**Reply by bl4ckswordsman** [reply]
Thanks!
I really hope it's doable in a clean way!

Doing a quick search it seems that Netdata implemented this in Go, so fingers crossed ([go.d/systemdunits](https://github.com/netdata/go.d.plugin/tree/master/modules%2Fsystemdunits)). I'd gladly help but I don't know any golang 🙁
